README.md

```diff
@@ -225,3 +189,33 @@ curl -s http://0.0.0.0:8080/block | jq
 ## Contribute
 
 Need help or want to contribute ideas or code? Head over to our [CONTRIBUTING](CONTRIBUTING.md) doc for more information.
+
+## Note for maintainers
+
+All the commits in this repo follow the Conventional Commits spec. When merging a PR, make sure 1/ to
+use squash merge and 2/ that the title of the PR follows the Conventional Commits spec. This will
+make squash the PR into one conventional commit.
+
+The history of commits will be used to generate the `CHANGELOG`. To do so, run yarn deploy on the master
+branch. This command will look at all the commits since the latest tag, bump the package version according
+to semver rules, and generate a new `CHANGELOG`.
+
+There might be special cases where you don't want to follow semver. In this case, run `yarn deploy -r {major,minor,patch}`.
+
+The above command, which only does local operations and doesn't push anything, will output more or
+less the following lines:
+
+``` bash
+$ yarn deploy
```

Comment:
"The above command" seems to refer to `yarn deploy -r {major,minor,patch}` but I think you mean `yarn deploy` here.